### DISCUSSION TITLE
Convert result of Attempt to a list

### DISCUSSION THREAD

**Discussion by MarloesAlb** [the-anylogic-company/AnyLogic-Pypeline#35] [Q&A] [answered]
I used the pyCommunicator to get some results from python. In my case, the result is a sequence, lets say
sequence = [2,5,3,6,4,1]

I would like to assign the priority of an agent according to this sequence. Each order has an orderID (1,2,3, etc)
For this, I wanted to use the index of the value in the sequence that is equal to the value of orderID.
Thismeans that the priority for order 1 is 6, as we find number 1 on index 6 in the sequence.
However, I cannot use get.(1) as I get the error ' The method get(int) is undefined for the type Attempt'

Does anyone know how I can solve this,or how to do this in another way??

**Comment by t-wolfeadam** [comment, marked as answer]
> as I get the error ' The method get(int) is undefined for the type Attempt'

As described [here](https://github.com/t-wolfeadam/AnyLogic-Pypeline/wiki/Usage-(basic)), you need to convert the Attempt object to a Java type by passing what class you want to convert it to (and you can use the 'shortcut' described in the linked-page to do this in one call). 

Also, using `get(1)` will return the value at index 1 (i.e., 5). If you want the reverse, you can use `indexOf`. For example:

```java
pyCommunicator.run("sequence = [2,5,3,6,4,1]");
ArrayList<Integer> seq = pyCommunicator.runResults(ArrayList.class, "sequence");
traceln(seq.get(1)); // prints 5
traceln(seq.indexOf(5)); // prints 1
```

**Reply by MarloesAlb** [reply]
Thanks for your answer!
